File: cases/taint/python/computed-property-positive/computed_property.py
def dfb_source():  # DFB-SOURCE: computed-property-input
    return "tainted"


def dfb_sink(value):  # DFB-SINK: computed-property-sink
    pass


class Holder:
    def __init__(self):
        self.alpha = "clean"
        self.beta = "clean"


def run():
    holder = Holder()
    key = "alpha"
    setattr(holder, key, dfb_source())  # DFB-WITNESS: computed-property-store
    dfb_sink(getattr(holder, key))

File: cases/taint/ruby/computed-property-positive/computed_property.rb
class Holder
  def initialize
    @alpha = "clean"
    @beta = "clean"
  end
end

def dfb_source # DFB-SOURCE: computed-property-input
  "tainted"
end

def dfb_sink(value) # DFB-SINK: computed-property-sink
end

def run
  holder = Holder.new
  key = "@alpha"
  holder.instance_variable_set(key, dfb_source) # DFB-WITNESS: computed-property-store
  dfb_sink(holder.instance_variable_get(key))
end
